services/src/main/java/com/chatbot/services/PubSubPullSubscriber.java

```diff
@@ -0,0 +1,127 @@
+package com.chatbot.services;
```

Comment:
I thought we decided to use push subscriber??

Reply:
Yeah, this is the original implementation, did not want to remove it in case later on someone wants to use it.